web/packages/design/src/utils/copyToClipboard.ts

```diff
@@ -15,13 +15,18 @@
  */
 
 /**
- * copyToClipboard copies text to clipboard.
+ * Copies text to clipboard.
+ * Works only in browser, not in Electron.
  *
  * @param textToCopy the text to copy to clipboard
  */
-export default function copyToClipboard(textToCopy: string): Promise<any> {
-  return navigator.clipboard.writeText(textToCopy).catch(err => {
-    // This can happen if the user denies clipboard permissions:
+export default async function copyToClipboard(
```

Comment:
As an alternative to this implementation, could we just extend the `utils/copyToClipboard` function to detect whether or not we're running in Electron and then based on that use either the `navigator.clipboard` or the `clipboard` module? If possible I think this would simplify things and would make it easier to port any features that use `copyToClipboard` to Connect without needing to pass it in as a prop

Reply:
While I was trying to answer Yassine's question, I found [a blogpost from Doyensec](https://blog.doyensec.com/2022/09/27/electron-api-default-permissions.html). They talk about `navigator.clipboard.readText` working by default and I was like… wait a minute, I thought it's not supported by Electron?

It turns out that clipboard in Electron works just fine, it's just that we followed their recommendations and turned off all permissions:

https://github.com/gravitational/teleport/blob/27b79a2684b43ee0d768b6f98f6da9bfe645e126/web/packages/teleterm/src/mainProcess/windowsManager.ts#L149-L154

Based on that blogpost, I think allowing `clipboard-sanitized-write` should be fine and we wouldn't need to change any code.